beeswax/openrtb/extension.proto

```diff
@@ -353,6 +353,25 @@ message UserExtensions {
   optional int32 cookie_age = 3;
   // GDPR-compliant hashed user_id
   optional string user_id_hashed = 8;
+  message CrossDeviceIds {
+    repeated string ids = 1;
+  }
+  message CrossDeviceVendorLinkedIds {
```

Comment:
Could you please add the "next available tag", thanks!

Reply:
we’re using the same internal tag for our public API, which means we’re not following the sequence.